shipment_advice/tests/test_shipment_advice.py

```diff
@@ -246,3 +246,54 @@ def test_shipment_advice_draft(self):
     def test_shipment_name(self):
         self.assertTrue("OUT" in self.shipment_advice_out.name)
         self.assertTrue("IN" in self.shipment_advice_in.name)
+
+    def test_auto_validate_shipment_advice(self):
```

Comment:
Please simplify this test.
Create two pickings with several move_lines and assign both pickings to the same shipment.advice.
Then validate just one picking and check if the shipment.advice is still open. 
After that validate the second picking and check if the shipment.advice is done.